examples/models/file/maps.py

```diff
@@ -1,8 +1,8 @@
 from bokeh.document import Document
 from bokeh.embed import file_html
-from bokeh.models import (BoxSelectTool, Circle, ColumnDataSource, GMapOptions,
-                          GMapPlot, Label, LinearAxis, MercatorTicker,
-                          MercatorTickFormatter, PanTool, WheelZoomTool,)
+from bokeh.models import (
+    BoxSelectTool, Circle, ColumnDataSource, GMapOptions, GMapPlot, Label, LinearAxis,
+    MercatorTicker, MercatorTickFormatter, PanTool, WheelZoomTool,)
```

Comment:
Trailing comma. Also there is lack of consistency between this and `glyphs.py` above.

Reply:
ah good catch, I'll look over the examples more carefully